src/components/Carousel.ts

```diff
@@ -165,32 +163,13 @@ export default defineComponent({
      */
     function updateSlideSize(): void {
       if (!viewport.value) return
-      let multiplierWidth = 1
-      let multiplierHeight = 1
-      transformElements.forEach((el) => {
-        const transformArr = parseTransform(el)
-
-        if (transformArr.length == 6) {
-          multiplierWidth *= transformArr[0]
-          multiplierHeight *= transformArr[3]
-        }
-      })
 
       // Calculate size based on orientation
+      const { width, height } = viewport.value.getBoundingClientRect()
```

Comment:
The idea was that if we have a number `height` specified, we don't need to call getBoundingClientRect which is a call that can force a reflow, so more performant

Reply:
The multiplier there was part of the animation check, without it the width inside a transform will not be properly calculated (you can see this in the playground animation)